### PR TITLE
[RO] Add `Upgrading to lazer`

### PR DESCRIPTION
**Pull request by 28edx**
Reuploading this because I forgot to change branch.
Translation for the `Upgrading to lazer` article.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+Cred că ai greșit jocul.
+
+## Note
+
+[^wine]: Folosind Wine.
+[^compatibility-mode]: DirectX cu modul de compatibilitate.
+[^dll]: Manual folosind fișiere `.dll`
+[^share-files]: Beatmap-urile și skin-urile vor impărții fișierele, și va salva din spațiul de stocare.
+[^gameplay-only]: Doar gameplay.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:285]:
```suggestion
[^gameplay-only]: Doar în gameplay.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+[^online]: Prin vizualizare online.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:286]:
```suggestion
[^online]: Prin obținere online de la server.
```
Though this should be updated in the original translation. It is no longer the case. Online retrieval is gone.
cc: @Walavouchey

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+[^normalisation]: Aduce culorile personalizate combo din beatmap la același nivel de luminozitate.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:287]:
```suggestion
[^normalisation]: Aduce culorile de combo personalizate din beatmap la același nivel de luminozitate.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+[^hold-for-hud]: Ține apăsat `Ctrl` pentru a vedea HUD-ul momentan atunci când este ascuns.
+[^offset-calibration-stable]: Ajustabil manual prin taste.
+[^offset-calibration-lazer]: Când reîncerci un beatmap, poți calibra offsetul bazat pe ultima ta încercare.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:290]:
```suggestion
[^offset-calibration-lazer]: Când reîncerci un beatmap, poți calibra offset-ul bazat pe ultima ta încercare.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+[^can-disable]: Poate fi dezactivat.
+[^note-lock]: Încă există, dar n-ar trebuii să încurce.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:292]:
```suggestion
[^note-lock]: Încă există, dar n-ar trebuii să încurce.
```
```suggestion
[^note-lock]: Încă există, dar nu ar trebui să încurce.
```

See: https://www.cumzic.ro/ar-trebui-sau-ar-trebuii/ (you make this mistake throughout the entire document)

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+[^score-reset-balance]: Scorurile vor fi resetate pentru a balansa jocul.
+[^score-reset-isolated]: Scorurile vor fi resetate, momentan izolate de scorurile stable.
+[^online-content]: Acces direct la majoritatea informațiilor online.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:295]:
```suggestion
[^online-content]: Acces în joc la majoritatea informațiilor de pe site.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+[^direct-supporter]: Prin osu!direct, doar osu!supporter.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:296]:
```suggestion
[^direct-supporter]: Prin osu!direct, doar ca suporter osu!.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+[^supporter]: Doar osu!supporter.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:297]:
```suggestion
[^supporter]: Doar ca suporter osu!.
```
See: https://crowdin.com/editor/osu-web/all/en-ro?view=comfortable&filter=basic&value=0#q=osu!supporter

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 9]
```diff
@@ -0,0 +1,312 @@
...
+[^soft-deletion]: Restabilește beatmap-uri șterse și alte date din setări. Ștergerile realizate vor fi definitiv atunci când ieși din joc.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:298]:
```suggestion
[^soft-deletion]: Restabilește beatmap-uri șterse precum și alte date din setări. Ștergerile realizate vor fi definitivate atunci când ieși din joc.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 10]
```diff
@@ -0,0 +1,312 @@
...
+[^multi-room-max]: Maxim 16 jucători.
+[^map-only]: Doar beatmap-ul.
+[^all-files]: Toate fișierele.
+[^editor-precise-rotation]: Lipsa funcției precise de rotație.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:302]:
```suggestion
[^editor-precise-rotation]: Lipsă funcție de rotație precisă.
```
These are tooltips and should be worded accordingly.

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 11]
```diff
@@ -0,0 +1,312 @@
...
+[^incompatibilities]: Niște funcții în editor vor cauza beatmap-urile să ruleze greșit în stable - va fi rezolvat în curând.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:303]:
```suggestion
[^incompatibilities]: Unele funcții în editor vor cauza beatmap-urile să ruleze greșit în stable - va fi rezolvat în curând.
```
Should probably be changed in the original version. (This is also no longer the case with the `.olz` and `.osz` format.) @Walavouchey would like your input on this

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 12]
```diff
@@ -0,0 +1,312 @@
...
+[^stable-chat]: Mesajele trimise pot dura până la 15 secunde să apară.
+[^countdown-timers-stable]: Setează un cronometru cu numărătoare inversă printr-o comandă, fără să înceapă automat.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:305]:
```suggestion
[^countdown-timers-stable]: Setează un cronometru cu numărătoare inversă printr-o comandă, fără a începe automat meciul.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 13]
```diff
@@ -0,0 +1,312 @@
...
+[^countdown-timers-lazer]: Setează un cronometru cu numărătoare inversă direct din UI pentru a începe meciul.
+[^queue-modes]: Activează pentru a permite oricărui jucător să pună beatmap-urile în lista de așteptare, adică „host rotate”.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:307]:
```suggestion
[^queue-modes]: Activează pentru a permite oricărui jucător să adauge beatmap-uri la coadă, adică „host rotate”.
```

File: wiki/Help_centre/Upgrading_to_lazer/ro.md [hunk 14]
```diff
@@ -0,0 +1,312 @@
...
+[^difficulty-adjust]: Poți schimba CS/AR/OD/HP unui beatmap direct din meniul de selecție cu modul de ajustare a dificultății.
```
Comment (NiceAesth) [inline wiki/Help_centre/Upgrading_to_lazer/ro.md:308]:
```suggestion
[^difficulty-adjust]: Poți schimba CS/AR/OD/HP-ul unui beatmap direct din meniul de selecție cu mod-ul „Difficulty Adjust”.
```
Mods are not currently localized.

**Review by NiceAesth — CHANGES_REQUESTED** [review]
Thanks for the translation, I have went through your changes and reviewed them. Please keep this feedback in mind for future contributions.

As a note, please review the existing term usage in the stable translation:
https://gist.githubusercontent.com/NiceAesth/eea5630fa03038fb06bc83274626dc6c/raw/4ecfb996496c016ca084d4bd5fdcc3d6cdb84c3d/ro.txt

as well as crowdin:
https://crowdin.com/translate/osu-web/all/en-ro

Additionally, if you require help there is a translation channel in the Romanian Discord server that we use for coordinating review efforts.

@entwastaken if you wish to double check, please feel free to do so.

**Comment by 28edx** [comment]
Wow, that's what I get for breezing trough it translating it, I deeply apologize for my unprofessionalism and for all of these grammatical errors. 

**I approved the suggestions.** 

Felt like I was back in high school hearing my teacher review someone's work. And also somehow failed to notice some english words not being separated with a „-”.

**Review by NiceAesth — APPROVED** [review]
looks ok

**Review by TicClick — APPROVED** [review]

File: wiki/Help_centre/Upgrading_to_lazer/ro.md (hunk 11 above)
Comment (TicClick) [reply wiki/Help_centre/Upgrading_to_lazer/ro.md]: (English line: https://github.com/ppy/osu-wiki/blob/master/wiki/Help_centre/Upgrading_to_lazer/en.md?plain=1#L303)

**Pull request merged**